webui/react/src/e2e/models/common/hew/Card.ts

```diff
@@ -1,37 +1,17 @@
-import {
-  BaseComponent,
-  CanBeParent,
-  NamedComponent,
-  NamedComponentArgs,
-} from 'e2e/models/common/base/BaseComponent';
-import { WorkspaceActionDropdown } from 'e2e/models/components/WorkspaceActionDropdown';
-
-import { DropdownMenu } from './Dropdown';
+import { BaseComponent, NamedComponent } from 'e2e/models/common/base/BaseComponent';
+import { DropdownMenu } from 'e2e/models/common/hew/Dropdown';
 
 /**
  * Represents the Card component in hew/src/kit/Card.tsx
  */
 export class Card extends NamedComponent {
-  override defaultSelector: string = ''; // must be provided
-  // provide an actionMenu with a Dropdown to use
-  static actionMenuSelector = '[aria-label="Action menu"]';
-
-  // default to a workspace dropdown to avoid non-null but this should be overriden if a dropdown exists
-  readonly actionMenu: DropdownMenu = new WorkspaceActionDropdown({
-    clickThisComponentToOpen: new BaseComponent({
-      parent: this,
-      selector: Card.actionMenuSelector,
-    }),
+  override defaultSelector = '[data-testid^="card"]';
+  readonly actionMenuContainer = new BaseComponent({
+    parent: this,
+    selector: '[aria-label="Action menu"]',
+  });
+  readonly actionMenu = new DropdownMenu({
+    clickThisComponentToOpen: this.actionMenuContainer,
     root: this.root,
   });
-
-  static withName<T extends Card>(
```

Comment:
we don't need a special initializer for this anymore. you can see we replace this with the new implementation of `attachment` from `NamedComponent`